vendor/bat-native-ads/src/bat/ads/internal/catalog/catalog_state.cc

```diff
@@ -72,14 +72,23 @@ Result CatalogState::FromJson(
     }
 
     // Day parts
-    for (const auto& day_part : campaign["dayParts"].GetArray()) {
-      CatalogDayPartInfo day_part_info;
+    for (const auto& daypart : campaign["dayParts"].GetArray()) {
+      CatalogDaypartInfo daypart_info;
 
-      day_part_info.dow = day_part["dow"].GetString();
-      day_part_info.start_minute = day_part["startMinute"].GetUint();
-      day_part_info.end_minute = day_part["endMinute"].GetUint();
+      daypart_info.days_of_week = daypart["dow"].GetString();
+      daypart_info.start_minute = daypart["startMinute"].GetUint();
+      daypart_info.end_minute = daypart["endMinute"].GetUint();
 
-      campaign_info.day_parts.push_back(day_part_info);
+      campaign_info.dayparts.push_back(daypart_info);
+    }
+    if (campaign_info.dayparts.empty()) {
+      CatalogDaypartInfo daypart_info;
+
+      daypart_info.days_of_week = "0123456";
+      daypart_info.start_minute = 0;
+      daypart_info.end_minute = 1440;
```

Comment:
Should this not be 1439

Reply:
nit: please use base::Time constants to calculate

Reply:
Fixed